tests/conftest.py

```diff
@@ -5,27 +5,40 @@
 import os
 import pathlib
 import shutil
+from dataclasses import dataclass, field
+from subprocess import CompletedProcess
 from typing import Generator
+from unittest.mock import patch
 
 import pytest
+import tomlkit
```

Comment:
tomlkit came along with hatch, do you think it's worth it to declare as a dep?
https://github.com/juftin/hatch-pip-compile/blob/f7d9dadf68e11f22bef289323784e23daaac19b2/requirements/requirements-test.txt#L177-L180

Reply:
We put things into dependencies when we use them, we don't just skip dependencies because something somewhere happens to pull it in **at the moment**. Hatch does not have a documented feature "provide tomlkit" and it could reasonably stop using that library any time.